src/Database/PostgreSQL/PQTypes.hs

```diff
@@ -59,6 +59,7 @@ module Database.PostgreSQL.PQTypes (
   ) where
 
 import Data.Functor.Identity
+import Database.PostgreSQL.PQTypes.Interval
```

Comment:
This change looks unnecessary. Did the sort imports script do that? Looks like it has a bug.

Reply:
Huh, right. It's only our repo's internal version that has this bug though, I fixed it some time ago: https://github.com/arybczak/haskell-dev-utils/commit/96d0578216a48fe3ba57b07a30bbf44021a6d1af

Reply:
Force pushed version without changes to this file.

Reply:
There are similar unnecessary changes to lots of other files, can you just rerun the fixed script on the whole repo perhaps?